README.md

```diff
@@ -185,7 +185,7 @@ https://web.microsoftstream.com/video/xxxxxxxx-aaaa-xxxx-xxxx-xxxxxxxxxxxx
 ```
 
 ### Title template
-The `-t` option allows users to input a template string for the output file names.
+The `-t` option allows users to input a custom or a template string for the output file names.
```

Comment:
I'm having trouble understanding this edit, could you please rephrase it?

My suggestion would be like 'The `-t` option allows user to specify a custom filename for the videos.'